src/assets.rs

```diff
@@ -111,7 +119,8 @@ impl AssetLoader for GameMetaLoader {
                 LoadedAsset::new(meta)
                     .with_dependencies(vec![start_level_path, main_menu_background_path])
                     .with_dependencies(locale_paths)
-                    .with_dependencies(font_paths),
+                    .with_dependencies(font_paths)
+                    .with_dependencies(vec![music_path]),
```

Comment:
We could move `music_path` into the vec created a couple lines above it so that we don't allocate another vector.